services/core-api/app/api/projects/information_requirements_table/models/information_requirements_table.py

```diff
@@ -47,6 +50,29 @@ def find_by_project_guid(cls, _id):
         except ValueError:
             return None
 
+    # TODO: fix the link variable to link tot he actual IRT page of a the project in question.
```

Comment:
Getting the correct link for IRT page in core-api. In my local: http://localhost:3000/pre-applications/53f4b01a-67da-477d-9b60-9880d4fc7b31/information-requirements-table/8b7c71fd-7679-4660-a3c8-96d436b3d8e1/intro-project-overview